compiler/src/Language/Bond/Codegen/Cpp/Grpc_cpp.hs

```diff
@@ -0,0 +1,34 @@
+-- Copyright (c) Microsoft. All rights reserved.
+-- Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+{-# LANGUAGE QuasiQuotes, OverloadedStrings, RecordWildCards #-}
+
+module Language.Bond.Codegen.Cpp.Grpc_cpp (grpc_cpp) where
+
+import Data.Monoid
+import Prelude
+import Data.Text.Lazy (Text)
+import Text.Shakespeare.Text
+import Language.Bond.Syntax.Types
+import Language.Bond.Codegen.TypeMapping
+import Language.Bond.Codegen.Util
+import qualified Language.Bond.Codegen.Cpp.Util as CPP
+
+-- | Codegen template for generating /base_name/_grpc.cpp containing
+-- definitions of helper functions and schema metadata static variables.
+grpc_cpp :: MappingContext -> String -> [Import] -> [Declaration] -> (String, Text)
+grpc_cpp cpp file _imports declarations = ("_grpc.cpp", [lt|
+#include "#{file}_reflection.h"
+#include "#{file}_grpc.h"
+#include <bond/core/exception.h>
```

Comment:
Do we directly generate code that throws Bond exceptions? I think we can omit this header.

Reply:
It's a holdover from Comm_cpp.hs, where it also may not be needed. I was erring on the side of caution but I'll look to remove it.